pkg/apis/resource/namedresources.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
+
+import "k8s.io/apimachinery/pkg/api/resource"
+
+// NamedResourcesResources is used in NodeResourceModel.
+type NamedResourcesResources struct {
```

Comment:
Try to keep comments in sync with the versioned types.

Reply:
Fixed. A diff between internal and external `types.go` and `namedresources.go` now only shows the necessary changes (dropping tags, etc.). The comments are identical.